README.md

```diff
@@ -245,6 +246,16 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ---
 
+## Laravel
+
+- [Official Documentation](https://laravel.com)
+- [Video Tutorials for Laravel](https://laracasts.com)
+- [Discussion Forum](https://laravel.io/forum)
+- [Serverless Deployment for Laravel](https://vapor.laravel.com)
+- [Micro Laravel Framework for APIs](https://lumen.laravel.com)
+- [Server Management for Laravel](https://forge.laravel.com)
+---
```

Comment:
Add a new line before the horizontal line

Reply:
@shahednasser have done so 👍